etc/ci/upload_nightly.sh

```diff
@@ -48,7 +48,10 @@ main() {
     elif [[ "${platform}" == "macbrew" ]]; then
         extension=tar.gz
         package=target/release/brew/*."${extension}"
-    elif [[ "${platform}" == "windows" ]]; then
+    elif [[ "${platform}" == "windows-gnu" ]]; then
```

Comment:
Will do! I'd been thinking they might end up quite different, but that can come later when we start working on snazzier packaging for MSVC.